### DISCUSSION TITLE
Read specific condition on/off from external script

### DISCUSSION THREAD

**Discussion by hongkongkiwi** [finit-project/finit#251] [General]
I have a need to read signal status from a bash script. e.g I have lte_connected and wifi_connected

is there a similar command to initctl -a status "service" but for conditions to know if they are set?

Right now I'm using: 
```
initctl -b -p cond dump | awk '{ print $4 }' | grep -q '^<usr/wifi_connected>'
``` 
This method is a bit awkward and involves two additional tools.

**Comment by troglobit** [comment]
Just closed issue #255 which implements `initctl cond get foo`. Hope that helps! :)

```
root@zero:~# initctl cond get foo
root@zero:~# initctl -v cond get foo
off
root@zero:~# initctl -v cond set foo
root@zero:~# initctl -v cond get foo
on
root@zero:~# echo $?
0
root@zero:~# initctl -v cond clr foo
root@zero:~# initctl -v cond get foo
off
root@zero:~# echo $?
1
```

**Reply by hongkongkiwi** [reply]
Very helpful thanks!
